examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/with-scaffolder.py

```diff
@@ -13,11 +13,8 @@
     ScaffoldRequest,
     scaffold_component_yaml,
 )
-from dagster_components.core.schema.objects import AssetSpecSequenceField
-from dagster_components.core.schema.resolvable_from_schema import (
-    ResolvableModel,
-    ResolvedFrom,
-)
+from dagster_components.resolved.core_models import AssetSpecSequenceField
+from dagster_components.resolved.model import ResolvableModel, ResolvedFrom
```

Comment:
the tense switch-up on from ...resolved.model import ResolvableModel is unfortunate